Lib/unittest/mock.py

```diff
@@ -1772,7 +1772,7 @@ def method(self, *args, **kw):
 
 _unsupported_magics = {
     '__getattr__', '__setattr__',
-    '__init__', '__new__', '__prepare__'
+    '__init__', '__new__', '__prepare__',
```

Comment:
Good catch! This would have been caught only if the test had used `__prepare__` or `__instancecheck__` since importing `_unsupported_magics` for test iteration would have still passed at https://github.com/python/cpython/blob/bf46a09dec372b85846216bd692d648dac08ac36/Lib/unittest/test/testmock/testmagicmethods.py#L356 ;)